Mathlib/RingTheory/HopfAlgebra.lean

```diff
@@ -90,3 +90,5 @@ instance toHopfAlgebra : HopfAlgebra R R where
 theorem antipode_eq_id : antipode (R := R) (A := R) = .id := rfl
 
 end CommSemiring
+
```

Comment:
Same comment as above: I'd rather replace `section CommSemiring` by the same `namespace` a few lines further below.

Reply:
Fixed in #14620.